consensus-types/types/signed_beacon_block.go

```diff
@@ -34,13 +32,13 @@ import (
 )
 
 var (
-	_ ssz.DynamicObject                   = (*SignedBeaconBlock)(nil)
-	_ constraints.SSZMarshallableRootable = (*SignedBeaconBlock)(nil)
+	_ ssz.DynamicObject                                                = (*SignedBeaconBlock)(nil)
+	_ constraints.SSZVersionedMarshallableRootable[*SignedBeaconBlock] = (*SignedBeaconBlock)(nil)
 )
 
 type SignedBeaconBlock struct {
-	Message   *BeaconBlock        `json:"message"`
-	Signature crypto.BLSSignature `json:"signature"`
+	*BeaconBlock `json:"message"`
```

Comment:
question: can we also change the json tag to something like `json:"beacon_block"` or will that break backwards compatibility?

Reply:
my theory is that we don't actually use the json tags since this type is never json (un)marshalled in practice. Should investigate this and if confirmed, I think we should remove the tags entirely to make this explicitly clear.

Reply:
confirmed and handled in https://github.com/berachain/beacon-kit/pull/2611